### PR TITLE
Add tree.py

### PR DESCRIPTION
**Pull request by beckyjackson**
Usage:
```
python3 tree.py <sqlite-database> <term-curie> > <html-file>
```

Creates an HTML page containing the tree structure of the term & its annotations. HTML is written to stdout.

The `sqlite-database` must be created by RDFTab (https://github.com/ontodev/rdftab.rs) and include 'statements' and 'prefixes' tables. The `term-curie` must use a prefix from the 'prefixes' table.

Note that this is __not__ an interactive tree. Clicking on other terms in the tree will take you to the PURL for that term.

### CODE REVIEW

File: scripts/tree.py [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+    children = ["ul", {"id": "children"}] + children
+    if len(children) == 0:
+        children = ""
+
+    hierarchy = ["ul", ["li", tree_label(data, treename, term_id), children]]
+    i = 0
+    parents = tree["parents"]
+    if parents:
+        node = parents[0]
+        if term_id != "NCBITaxon:1":
```
Comment (jamesaoverton) [inline scripts/tree.py:97]: We need to generalize this root. Suggestions?

**Comment by jamesaoverton** [comment]
It's a bit tricky, but I would like the `rel` attributes to be PURLs, while (most of?) the `href` attributes are relative links. That should allow the tree to be interactive (relative links), while still supporting RDFa translations (absolute PURLs). It's not clear to me what's happening in the current implementation.

Some unit tests would help with this.

**Comment by beckyjackson** [comment]
All of the links right now are absolute PURLs since this was just the first pass. I can change the `rel` from the current state (CURIE) to the full IRI.

What should the relative links be? `?curie=[curie]`?

**Comment by jamesaoverton** [comment]
Sorry, I misspoke: `rel`s can be CURIEs, that's fine. The RDFa tools will handle that.

I think I prefer `?id=`

File: scripts/tree.py (hunk 1 above)
Comment (beckyjackson) [reply scripts/tree.py]: I don't think we need this `if term_id != ...` statement; I removed it and it doesn't look like there's any unintended behavior, but what do you think?

**Comment by beckyjackson** [comment]
No worries. I've updated the hrefs to be `?id=`, except in the case of the subject IRI that appears in the header. That's still the PURL.

File: scripts/tree.py (hunk 1 above)
Comment (jamesaoverton) [reply scripts/tree.py]: I'm not sure. It's a recursive loop that hung on me a few times. Get rid of it and we'll see! :^)

**Comment by jamesaoverton** [comment]
@beckyjackson Please fix the label bug I showed you, then we'll merge.

**Comment by beckyjackson** [comment]
Should we just remove `rdfs:label` entirely from the annotations, since it's already displayed at the top of the page?

**Comment by beckyjackson** [comment]
This is ready to merge

**Pull request merged**